package/src/index.ts

```diff
@@ -1,11 +1,33 @@
 import { NitroModules } from "react-native-nitro-modules";
 import type {
   AppUser,
+  AssetAccount as NitroAssetAccount,
+  AssetAccountQuery,
+  FiatAsset,
+  FiatAssetQuoteRequest,
   LinkedAccount,
+  MarketAssetQuoteRequest,
+  MarketTradeAsset,
+  NothingAsset,
+  NothingAssetQuoteRequest,
+  OtherAsset,
   PresentationBackground,
   PresentationStyle,
   RNCandle,
   Service,
+  TradeAsset as InternalTradeAsset,
+  TradeAssetQuoteRequest,
+  TradeQuery,
+  TransportAsset,
+  TransportAssetQuoteRequest,
+  LegalAccountKind,
+  WireDetails,
+  ACHDetails,
+  TradeState,
+  MerchantCounterparty,
+  UserCounterparty,
+  ServiceCounterparty,
+  Counterparty as InternalCounterparty,
 } from "./specs/RNCandle.nitro";
 
 export class CandleClient {
```

Comment:
#nth this class should be in its own file, `index.ts` should only contain re-exports from other files